oracle-plugin/docs/Oracle-batchsink.md

```diff
@@ -38,6 +38,38 @@ will be passed to the JDBC driver as connection arguments for JDBC drivers that
 
 **Default Batch Value:** The default batch value that triggers an execution request.
 
+
+Data Types Mapping
+----------
+
+	| Oracle Data Type               | CDAP Schema Data Type        | Comment                |
+	| ------------------------------ | ---------------------------- | ---------------------- |
+	| VARCHAR2                       | Type.STRING                  |                        |
```

Comment:
similar comment in other PR about using the names that people see in the UI.

Reply:
Changed.